tendrl/ceph_integration/objects/cluster/__init__.py

```diff
@@ -0,0 +1,22 @@
+from tendrl.commons.etcdobj import EtcdObj
+from tendrl.ceph_integration import objects
+
+
+class Cluster(objects.CephIntegrationBaseObject):
+    def __init__(self, cluster_id=None, name=None,
+                 status=None, *args, **kwargs):
+        super(Cluster, self).__init__(*args, **kwargs)
+
+        self.value = 'clusters/%s'
```

Comment:
Do you realize that if we add attributes like say "Volumes" to this cluster object, it will overwrite the actual "Volume" object at /clusters/id/Volumes? 

To avoid such a huge impact, I am asking for the change